lib/rails/generators/rake_migrations/templates/migration_rails_on_and_before_v4.rb

```diff
@@ -1,6 +1,6 @@
 class CreateRakeMigrationsTable < ActiveRecord::Migration
   def self.up
-    create_table :rake_migrations do |t|
+    create_table :rake_migrations, id: false do |t|
       t.string :version
     end
```

Comment:
Why are we not handling `down` change here ?

Reply:
we are dropping the table in `down`
https://github.com/Bizongo/rake-migrations/blob/ebb64b334c2ccf513ac96532a59342b6dfe17b64/lib/rails/generators/rake_migrations/templates/migration_rails_on_and_before_v4.rb#L8